core/src/main/java/org/arquillian/smart/testing/TestSelection.java

```diff
@@ -14,23 +16,28 @@
 
     private final String className;
 
-    private final Collection<String> types; // TODO or strategy instead (misleading name)
+    private final Collection<String> appliedStrategies;
 
-    public TestSelection(Path location, String... type) {
-        this(new ClassNameExtractor().extractFullyQualifiedName(location), type);
+    private final List<String> testMethodNames;
+
+    public static final TestSelection NOT_MATCHED = new TestSelection("");
```

Comment:
I can't see, where is it used?

Reply:
In a real implementation - check this PR: https://github.com/arquillian/smart-testing/pull/309
https://github.com/arquillian/smart-testing/pull/309/commits/e6314b83a7a4756d23b566994d9f99edc02934d9#diff-b179c0d6894a22602721a54a8efc1d54R33

Reply:
According to IDEA nowhere. Probably some left-over @MatousJobanek?

Reply:
Yeah, sorry for that - I should have put it into the next PR - where it is really used. I just introduced it along with the other changes, so using the command `git add core/*` I put it into this PR...